crates/driver/src/infra/liquidity/config.rs

```diff
@@ -124,7 +124,7 @@ impl UniswapV3 {
     pub fn uniswap_v3(network: &eth::NetworkId) -> Option<Self> {
         Some(Self {
             router: deployment_address(contracts::UniswapV3SwapRouter::raw_contract(), network)?,
-            max_pools_to_initialize: 50,
+            max_pools_to_initialize: 100,
```

Comment:
Is there a reason we want to increase this again?

Reply:
This is the default in the current infra so I wanted to make it consistent.